### PR TITLE
fix: improve error message when providing an invalid username

### PR DESCRIPTION
**Pull request by cristiand391**
### What does this PR do?
Improves error message to show which username (devhub username/username) is causing problems when trying to authenticate.

### What issues does this PR fix or reference?
@W-9464074@

### CODE REVIEW

File: messages/core.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   "OrgDataNotAvailableErrorAction2": "Use `sfdx force:config:set` to update the defaultusername.",
   "OrgDataNotAvailableErrorAction3": "Use `sfdx force:org:create` to create a new org.",
   "OrgDataNotAvailableErrorAction4": "Use `sfdx force:auth` to authenticate an existing org.",
-  "NamedOrgNotFound": "No AuthInfo found for name %s",
+  "NamedOrgNotFound": "No AuthInfo found for %s %s",
```
Comment (jshackell-sfdc) [inline messages/core.json:15]: Let's improve the message slightly. How about:

"No authorization information found for %s %s"

**Review by mshanemc — APPROVED** [review]

**Review by WillieRuemmele — APPROVED** [review]

**Comment by WillieRuemmele** [comment]
before/after
```
 ➜  sfdx force:user:display                                      
ERROR running force:user:display:  No AuthInfo found for name dev
 ➜  sfdx force:user:display -u abc         
ERROR running force:user:display:  No AuthInfo found for name abc
```
After:
```
 ➜  sfdx force:user:display
ERROR running force:user:display:  No authorization information found for devhub username dev
 ➜  sfdx force:user:display -u abc                                         
ERROR running force:user:display:  No authorization information found for username abc
```

File: messages/core.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
   "OrgDataNotAvailableErrorAction2": "Use `sfdx force:config:set` to update the defaultusername.",
   "OrgDataNotAvailableErrorAction3": "Use `sfdx force:org:create` to create a new org.",
   "OrgDataNotAvailableErrorAction4": "Use `sfdx force:auth` to authenticate an existing org.",
-  "NamedOrgNotFound": "No AuthInfo found for name %s",
+  "NamedOrgNotFound": "No authorization information found for %s %s",
```
Comment (jshackell-sfdc) [inline messages/core.json:15]:
```suggestion
  "NamedOrgNotFound": "No authorization information found for %s %s.",
```

File: messages/core.json [hunk 3]
```diff
@@ -12,7 +12,7 @@
...
   "NoAliasesFound": "Nothing to set",
```
Comment (jshackell-sfdc) [inline messages/core.json:16]:
```suggestion
  "NoAliasesFound": "Nothing to set.",
```

File: messages/core.json [hunk 4]
```diff
@@ -12,7 +12,7 @@
...
   "InvalidFormat": "Setting aliases must be in the format <key>=<value> but found: [%s]",
```
Comment (jshackell-sfdc) [inline messages/core.json:17]:
```suggestion
  "InvalidFormat": "Setting aliases must be in the format <key>=<value> but found: [%s].",
```

**Comment by jshackell-sfdc** [comment]
@cristiand391 , I made a teeny edits (just added a period) to some error messages. Okay?  Sorry I missed that earlier.

**Pull request merged**